tests/test_catalog.py

```diff
@@ -112,6 +112,75 @@ def test_search_live(catalog_live):
     assert search_results["type"] == "FeatureCollection"
 
 
+@pytest.mark.parametrize(
+    "usage_type,result,result2",
+    [
+        (["DATA"], "DATA", ""),
+        # (["ANALYTICS"], "ANALYTICS", ""), #TODO
```

Comment:
is this a TODO because the mock catalog currently only returns "DATA" or "DATA and ANALYTICS"?

Reply:
nice, was just fixing this (now all 3 options are tested), can you approve again please, somehow it automatically rejects reviews now when adding new commits.